dev-python/pychroot/pychroot-9999.ebuild

```diff
@@ -10,8 +10,8 @@ if [[ ${PV} == *9999 ]] ; then
 	EGIT_REPO_URI="https://github.com/pkgcore/pychroot.git"
 	inherit git-r3
 else
-	KEYWORDS="~amd64 ~x86"
-	SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+	KEYWORDS="~amd64 ~arm ~arm64 ~x86"
+	inherit pypi
```

Comment:
I think putting inherit before KEYWORDS is better for consistently?

Reply:
It follows the style of the other branch.